ll-property-image.html

```diff
@@ -298,63 +320,49 @@
     },
 
     ready: function() {
-      this.imgTitle = this.imgTitle || this.imgName;
-      this.imgDescription = this.imgDescription || '';
+      this.title = this.title || this.name;
+      this.description = this.description || '';
       this.isDefaultImage = this.isDefaultImage || false;
```

Comment:
These defaults seems weird to me because most of them should be set in the property declaration. Was there a specific reason for this?